Vagrantfile

```diff
@@ -43,6 +41,8 @@ Vagrant.configure("2") do |config|
   # Redirecting STDERR to /dev/null because Minikube print download progress
   # for the images to STDERR which clutters up the Vagrant output w/ error output!
   config.vm.provision :shell, privileged: false, inline: "sudo -g docker minikube start 2>/dev/null"
+  # Install everything from secureCodeBox:
+  config.vm.provision :shell, privileged: false, inline: "sudo -g docker /vagrant/bin/install.sh --all"
```

Comment:
sudo could be avoided if docker groups are used, but I'm not sure if that makes sense in this context

Reply:
No couldn't be avoided. All provisioners run in the same login session. So at this point the vagrant user was added to the group docker (see install-minikube.sh#83), but this has no effect until the user relogin. This is not possible here. So the workarround is (as documented in line 40) to use `sudo -g` to execute the command with the docker group. So this does exactly what you recommend.